openproblems/tasks/_cell_cell_communication/_common/metrics/odds_ratio.py

```diff
@@ -4,13 +4,17 @@
 import numpy as np
 
 
+def _sigmoid_transform(x):
+    return 1 - 1 / (1 + x / 2)
```

Comment:
@dbdimitrov are you happy with this scaling?

Reply:
Yep, tested on the preliminary benchmark results and it looked good - i.e. was able to easily distinguish odds of 8 from 2 :)